src/App.js

```diff
@@ -1,26 +1,13 @@
 import React from 'react';
-import logo from './logo.svg';
+import Home from './home'
 import './App.css';
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+    return(
+        
+        <Home/>
+      
+    );
+    }
 
-export default App;
+    export default App;
```

Comment:
Ten en cuenta el comentario de la indentación que, además de mejorar la legibilidad, hace que el código se vea mucho más prolijo:
```suggestion
export default App;
```